data/json/recipes/recipe_deconstruction.json

```diff
@@ -554,6 +554,30 @@
     "qualities": [ { "id": "CUT", "level": 1 } ],
     "components": [ [ [ "leather", 2 ] ], [ [ "string_6", 2 ] ] ]
   },
+  {
+    "result": "battery_car",
```

Comment:
There is an existing (reversible) crafting recipe for this item, with different components.

Reply:
It would enable you to transmute lead into sulphuric acid, but yes that would work.

Reply:
Ahhh. I see. Gonna go putter around some more with git blame and get the construction/decon values to sit happy with each other then.